tests/sentry/search/events/builder/test_metrics.py

```diff
@@ -2724,14 +2724,61 @@ def test_on_demand_map_with_multiple_selected(self):
             query=query_str,
             selected_columns=["eps()", "epm()", "not_on_demand"],
             config=QueryBuilderConfig(
-                on_demand_metrics_enabled=True, on_demand_metrics_type=MetricSpecType.SIMPLE_QUERY
+                on_demand_metrics_enabled=True, on_demand_metrics_type=MetricSpecType.DYNAMIC_QUERY
```

Comment:
Why this change?

Reply:
We're the only ones who use multiple aggregates (alerts is only 1 aggregate I believe), these multiple selected will only ever be called with dynamic query, so it's more appropriate for this to be `DYNAMIC_QUERY`